lib/protocol/packets/AuthSwitchPacket.js

```diff
@@ -0,0 +1,13 @@
+module.exports = AuthSwitchPacket;
+function AuthSwitchPacket(options) {
+  options = options || {};
+  this.scrambleBuff = options.scrambleBuff;
```

Comment:
Since this is greenfield, I'm just changing these props to have the same name as in the protocol docs for sanity.